website/docs/tutorials/08-interacting-with-starknet/07-multicall.md

```diff
@@ -0,0 +1,52 @@
+# Multicall
+
+## Overview
+Multicall allows executing multiple calls while providing [atomicity](https://en.wikipedia.org/wiki/Atomicity_(database_systems)). If one call fails, the entire operation is aborted. Use it to prevent leaving your system in an inconsistent state. 
```

Comment:
```suggestion
Multicall executes multiple calls as a single transaction. If one call fails, the entire operation is aborted. Use it to prevent leaving your system in an inconsistent state. 
```
Atomicity does not relate to consistency. Atomicity relates to not beaing interleaved by another transaction. https://en.wikipedia.org/wiki/ACID

Reply:
Consistency is a property of an application that is achieved using Atomicity and Isolation, hence Atomicity is related to Consistency.

Atomicity is more about abortability.